src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -501,15 +501,21 @@ export class BrowsingContextImpl {
     });
 
     this.#cdpTarget.cdpClient.on('Page.javascriptDialogClosed', (params) => {
+      const accepted = params.result;
+
       this.#eventManager.registerEvent(
         {
           type: 'event',
           method: ChromiumBidi.BrowsingContext.EventNames.UserPromptClosed,
           params: {
             context: this.id,
-            accepted: params.result,
-            // Cast empty string to undefined
-            userText: params.userInput ? params.userInput : undefined,
+            accepted,
+            ...(accepted
```

Comment:
I think we can just do 
userText: accepted && params.userInput ? params.userInput : undefined,

Reply:
Done